templates/resources/ec2_vpc_endpoint.md.tmpl

```diff
@@ -0,0 +1,35 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### Basic usage
+To create a VPC
+{{ tffile (printf "examples/resources/%s/vpc_endpoint_basic.tf" .Name)}}
+
+### Interface Endpoint Type
+To create a VPC Endpoint with Interface type
+{{ tffile (printf "examples/resources/%s/vpc_endpoint_interface_type.tf" .Name)}}
+
+### Gateway Endpoint Type
```

Comment:
this should be `Gateway Load Balancer Endpoint Type` in order to distinguish between three types:

`Interface | Gateway | GatewayLoadBalancer`

Reply:
Modified